### DISCUSSION TITLE
Error: Out of range value at section

### DISCUSSION THREAD

**Discussion by paolo2504** [GrandOrgue/grandorgue#1025] [Help]
hi all, i'm getting the above error:
Error: Out of range value at section 'General001' entry 'StopNumber010': -10
in the HW1 .organ file below attached.
any hint appreciated.

[Wildervank_HW1.zip](https://github.com/GrandOrgue/grandorgue/files/8043032/Wildervank_HW1.zip)

**Comment by oleg68** [comment]
This is an error in the ODF file: 
```
[General001]
...
StopManual010=000
StopNumber010=-010
```

But the `[Manual000]` (PEDAL) has only 9 stops with numbers from 1 to 9. So the values 10 and -10 are out of range.

**Comment by willeke1234** [comment]
Tip: If you want to use the Walcker Wildervank organ then use the link on the Sample Sets wiki page. It has a GrandOrgue ODF addon.

**Reply by paolo2504** [reply]
thanks willeke1234 i found it and give a try.
